docs/release-notes/cadl-typespec-migration.md

```diff
@@ -0,0 +1,69 @@
+# Cadl to TypeSpec rename completed in March 2023 release
+
+As you may recall from our previous email to partners, we are renaming the product as it becomes more mature, stable and one step closer to release.
```

Comment:
I would remove this about emailing partners.

Reply:
Doh!